### ISSUE TITLE
Reproducing model effects

### ISSUE THREAD

**Issue by 2076703026** [yilmazkorkmaz1/SSDiffRecon#3]
Hello, I have successfully reproduced your code using the dataset you provided. But there is a problem, I made my own training set and test set for training and testing, and the model I got using my own test set is a mess, but using the test set provided by you the result is normal, and I am using the mask training and testing dataset provided by you, can you explain in detail about your dataset creation?
Thank you very much, it's an honour to reproduce your project!

**Comment by yilmazkorkmaz1** [comment]
Hi, 

The model needs tfrecords format for the dataset, you can prepare it using dataset_tool.py. Please consider using float or int options for complex and only magnitude images respectively. Please refer to the stylegan2 codebase for more details https://github.com/NVlabs/stylegan2?tab=readme-ov-file#preparing-datasets/
